src/app/cube/new-home/help/teach-now/teach-now.component.html

```diff
@@ -1 +1,20 @@
-<p>teach-now works!</p>
+<div class="wrapper">
+    <div #topicScroll *ngIf="width > 750" class="topics">
+        <div class="shadow left" aria-hidden="true"></div>
+        <button *ngFor="let topic of topics" class="topic-btn" (activate)="selectTopic(topic._id)" [ngClass]="{'selected': selectedTopic === topic._id}">{{ topic.name }}</button>
```

Comment:
Would it be better to have buttons on the left and right side of the screen? Scrolling through the topic buttons felt a bit awkward.

Reply:
I understand how it would feel awkward, was following the mocks.  Let's put a pin in it (I'll add a comment in the html) if it becomes a usability issue in the future